index.html

```diff
@@ -0,0 +1,375 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="queries.css" />
+    <script
+      type="module"
+      src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
+    <script
+      nomodule
+      src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js"></script>
+    <title>Online School</title>
+  </head>
+  <body>
+    <!-- main page header -->
+    <div class="container">
+      <header class="header sticky">
+        <nav class="navigation">
+          <ul class="top-nav">
+            <li class="top-nav--element">
+              <a href="#"
+                ><ion-icon class="facebook-icon" name="logo-facebook"></ion-icon
+              ></a>
+            </li>
+            <li class="top-nav--element">
+              <a href="#"
+                ><ion-icon class="twitter-icon" name="logo-twitter"></ion-icon
+              ></a>
+            </li>
+            <li class="top-nav--element"><a href="#">English</a></li>
+            <li class="top-nav--element"><a href="#">My Page</a></li>
+            <li class="top-nav--element"><a href="#">Logout</a></li>
+          </ul>
+          <a href="index.html">
+            <img
+              class="logo"
+              src="./img/nav-logo.jpg"
+              alt="website logo"
+              width="100px" />
+          </a>
+          <button class="btn-mobile-nav">
+            <ion-icon class="btn__nav btn__open" name="menu-outline"></ion-icon>
+            <ion-icon
+              class="btn__nav btn__close"
+              name="close-outline"></ion-icon>
+          </button>
+          <ul class="navigation__list">
+            <li class="navigation__list--links">
+              <a href="about.html" style="color: #e32816; border-color: #e32816"
+                >About</a
+              >
+            </li>
+            <li class="navigation__list--links"><a href="#">Home</a></li>
+            <li class="navigation__list--links"><a href="#">Join</a></li>
+            <li class="navigation__list--links"><a href="#">Sponsor</a></li>
+            <li class="navigation__list--links"><a href="#">News</a></li>
+            <button class="navtigation--btn">Campaign</button>
+          </ul>
+        </nav>
+      </header>
+
+      <!-- HOME Page Section -->
+      <section class="home__section">
+        <h1 class="home__primary--header">
+          your childrens experience <br />
+          a better eudcation
+        </h1>
+        <p class="home__description">
+          your childrens will have a better learing experience With Us
+        </p>
+        <p class="home__date">2015.10.15(THU)</p>
+        <p class="home__copy-right">@ Netional School of online</p>
```

Comment:
- [ ] This section should contain a small line of text, the page title, a block of text, a medium line of text in bold letters, and a smaller line of text. But you don't have a small line of the text before the title, please kindly add it 🙃
- [ ] I think it could be better to avoid some typo errors like `CHILDRENS`, ` EUDCATION` in your title, kindly fix that 🙃 
- [ ] Also the font size of the media links is so big, I think it could be better if you reduce a little bit the size 😉 

Your design
<img width="930" alt="Screen Shot 2023-02-09 at 13 42 45" src="https://user-images.githubusercontent.com/74501523/217794279-3f2f3fbd-47a7-47d1-91b5-3e89770c73b0.png">

Cindy's design
<img width="930" alt="Screen Shot 2023-02-09 at 13 55 59" src="https://user-images.githubusercontent.com/74501523/217794345-974c5bd6-832d-4320-a421-f3b69d82120c.png">